docs/ui-ux/README.md

```diff
@@ -1,16 +1,80 @@
+
+> Be certain to not over-promise on UI/UX requirements in the gameplan document. CSE teams should **not** agree to delivering production-ready user interface applications. The engineering complexities and requirements are far too much for an engagement. Committing to proof-of-concept or minimum-valuable-product applications is significantly more achievable within the timeline of a single (or even multiple) engagements.
+
+User interface application design documents cover a variety of aspects. Before detailling the technical implementation requirements, the design document should focus on the intended user and the proposed solutions. Commonly referred to as "user stories", this section should contain prose-like paragraphs describing a literal user of the application attempting to do something with it. Generally, this section appears as a list with the personas and stories listed together. Personas are generally used more than once for each of the applications intended user paths. Stories should specific to the persona.
+
+For example, here are some user stories and personas for an application for searching through a data set of images tagged with common descriptors.
+
+1. Persona: A user that can see and use a mouse without assistance.
+   Story: I start by accessing the application in my browser. I log in using my corporate account, and then see a search box and a list of images. I start typing "dog" into the search box. The images begin shuffling around, with ones tagged "dog" appearing at the top.
+1. Persona: A user that requires an audio-aide for browsing the web since they cannot see.
+   Story: I start by accessing the application in my browser and with my audio-aide enabled. I am immediately focussed on the authentication form and enter my corporate information. Once I land on the main page, I'm focussed on the search box. I tab to the image list and use my aide to listen to the first 10 images displayed. I tab back to the search box and type "dog". I then tab to the image list against and listen to the list of images, now all returning some form of my search "dog".
+
+User stories help you and your team understand application and feature requirements. In application design, the user drives the features. By completely understanding your user, you in turn, can completely understand what you need to build.
+
+Following the user stories, the design document should detail the accessibility requirements of the application. **Accessibility is never optional**. Microsoft has made a public commitment to deliver No matter who you determine your users are, there is a strict set of minimum requirements for all [Microsoft applications](https://www.microsoft.com/en-us/accessibility).
+
+> If you haven't already, download the [Accessibility Insights](https://accessibilityinsights.io/) browser tool. You can use this, in combination with other accessibility-related tools, to easily check your application.
```

Comment:
Also I believe there is a node module or similar app which one can put as part of the CI/CD process to report out accessibility issues and how accessible a site is every time it is built.  Maybe look that up and include it here as well for automation.

Reply:
That will be good for the accessibility document